src/ripple/app/misc/HashRouter.h

```diff
@@ -84,12 +84,18 @@ class HashRouter
             flags_ |= flagsToSet;
         }
 
-        void swap (std::set <PeerShortID>& other)
+        std::set <PeerShortID>&& peers()
```

Comment:
On second, thought, renamed to `peekPeers`, return a reference, and let the caller do the move so nobody is surprised.